tests/neg/i6762.scala

```diff
@@ -0,0 +1,5 @@
+import scala.quoted.{_, given}
+
+type G[X]
+case class Foo[T](x: T)
+def f(word: String)(given QuoteContext): Expr[Foo[G[String]]] = '{Foo(${word.toExpr})} // error
```

Comment:
Is `given QuoteContext` needed here? The original issue did not have this implicit parameter.

Reply:
Yes, there is a quote. the original issue is outdated.

Reply:
Without it there is an extra error about the missing QuoteContext